### PR TITLE
Bringing ❤️️ to the readme

### PR DESCRIPTION
**Pull request by noway421**
A nicer reame

### CODE REVIEW

**Comment by feikname** [comment]
Oh, nice!

File: README.md [hunk 1]
```diff
@@ -1,43 +1,55 @@
...
-<br>
 
-> **What about 0.76 support**
->
-> Working with multiple versions is a pain. 0.76 will be suported in the
-> future only.
+#### How long will it take for the first release?
 
-<br>
+March 2016 probably.
```
Comment (VelocityRa) [inline README.md]: Er, don't you mean 2017?

Comment (feikname) [inline README.md]: @VelocityRa Oops!

Looks like we didn't got used to 2017 yet :P.

Going to fix it right now!

Comment (noway421) [inline README.md]: it came too soon

**Pull request merged**